### PR TITLE
Add vertex_ai_feature_store_iam_policy notebook.

### PR DESCRIPTION
**Pull request by ethan-gordon**
**REQUIRED:** Add a summary of your PR here, typically including why the change is needed and what was changed. Include any design alternatives for discussion purposes.

<br>
Add vertex_ai_feature_store_iam_policy notebook. This provides a tutorial on how to configure IAM Policy for Feature Store online serving resources.
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [x] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

<br>

3. If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/community-content/CODEOWNERS) file under the `Community Content` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

**Review by felipeliliti — APPROVED** [review]

**Review by felipeliliti — APPROVED** [review]
Ficou bom

**Comment by gericdong** [comment]
@ethan-gordon: please follow the notebook template: https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb

File: notebooks/official/feature_store/vertex_ai_feature_store_iam_policy.ipynb [hunk 1]
```diff
@@ -0,0 +1,612 @@
...
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "tvgnzT1CKxrO"
+      },
+      "source": [
+        "## Overview\n",
+        "\n",
+        "This tutorial demonstrates how to configure an IAM Policy to control access to resources and data stored within `Vertex AI Feature Store`.\n",
```
Comment (kittyabs) [inline notebooks/official/feature_store/vertex_ai_feature_store_iam_policy.ipynb:67]: remove `..`. from Vertex AI Feature Store. Should not be in code font.

File: notebooks/official/feature_store/vertex_ai_feature_store_iam_policy.ipynb [hunk 2]
```diff
@@ -0,0 +1,612 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "SbmM4z7FOBpM"
+      },
+      "source": [
+        "<div class=\"alert alert-block alert-warning\">\n",
+        "<b>⚠️ The kernel is going to restart. Please wait until it is finished before continuing to the next step. ⚠️</b>\n",
```
Comment (kittyabs) [inline notebooks/official/feature_store/vertex_ai_feature_store_iam_policy.ipynb:142]: remove "Please" and contract "it is"

- Wait until it's finished...

File: notebooks/official/feature_store/vertex_ai_feature_store_iam_policy.ipynb (hunk 1 above)
Comment (ethan-gordon) [reply notebooks/official/feature_store/vertex_ai_feature_store_iam_policy.ipynb]: Fixed.

File: notebooks/official/feature_store/vertex_ai_feature_store_iam_policy.ipynb (hunk 2 above)
Comment (ethan-gordon) [reply notebooks/official/feature_store/vertex_ai_feature_store_iam_policy.ipynb]: Fixed.

File: notebooks/official/feature_store/vertex_ai_feature_store_iam_policy.ipynb [hunk 3]
```diff
@@ -0,0 +1,708 @@
+{
```
Comment (gericdong) [inline notebooks/official/feature_store/vertex_ai_feature_store_iam_policy.ipynb:1]: <p>Update the links</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2913/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (ethan-gordon) [reply notebooks/official/feature_store/vertex_ai_feature_store_iam_policy.ipynb:1]: <p>Fixed. The notebook should appear in main once the PR is merged.</p>

**Review by gericdong — APPROVED** [review]

**Pull request merged**